decidim-proposals/app/cells/decidim/proposals/proposal_cell.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+require "cell/partial"
+
+module Decidim
+  module Proposals
+    # This cell renders the proposal card for an instance of a Proposal
+    # the default size is the Medium Card (:m)
+    class ProposalCell < Decidim::Proposals::ViewModel
+      include Cell::ViewModel::Partial
+      include Messaging::ConversationHelper
+
+      delegate :user_signed_in?, :current_user, to: :parent_controller
+
+      property :title
+      property :state
+      property :answered?
+      property :withdrawn?
+
+      def show
+        cell card_size, model, @options
+      end
+
+      def footer
```

Comment:
If we're just redirecting to `proposal_m`, isn't this unused? 😕

Reply:
Ooh! yes good catch, I'll remove them